stories/CodeOverview.stories.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { Store } from 'redux';
+import { storiesOf } from '@storybook/react';
+
+import { LinterMessage, actions } from '../src/reducers/linter';
+import longFileSample from './fixtures/long-file-sample';
+import CodeOverview from '../src/components/CodeOverview';
+import CodeView from '../src/components/CodeView';
+import configureStore from '../src/configureStore';
+import { JS_SAMPLE } from './CodeView.stories';
+import { createInternalVersion } from '../src/reducers/versions';
+import {
+  createFakeExternalLinterResult,
+  fakeVersion,
+  fakeVersionFile,
+  fakeVersionEntry,
+} from '../src/test-helpers';
+import { newLinterMessageUID, renderWithStoreAndRouter } from './utils';
+
+const render = ({
+  content = JS_SAMPLE,
+  messages = [],
+  store = configureStore(),
+}: {
+  store?: Store;
+  content?: string;
+  messages?: Partial<LinterMessage>[];
+} = {}) => {
+  let version = createInternalVersion(fakeVersion);
+
+  if (messages.length) {
+    const path = 'background.js';
+    version = createInternalVersion({
```

Comment:
You could just create this version as the default version (i.e., at line 29, via a `const` instead of a `let`), instead of having it inside this `if` block. It would be fine for all stories, and would allow you to have only the special logic for the `with messages` story inside this `if`. I was a bit confused about why you had to create a different `version` in this block just for the `with messages` story and then realized that this version could work for all stories.

I think that only including the logic that is specific to the `with messages` story in this `if` block makes it a bit easier to understand.